src/Administration/Resources/app/administration/src/module/sw-review/page/sw-review-detail/index.js

```diff
@@ -116,6 +150,10 @@ Component.register('sw-review-detail', {
 
         onSaveFinish() {
             this.isSaveSuccessful = false;
+        },
+
+        onCancel() {
```

Comment:
why is this necessary? Please revert this change if it is only cosmetic.

Reply:
Otherwise the ESCAPE key cannot properly be mapped, see:
https://github.com/shopwareBoostDay/platform/pull/156/files#diff-b5bd3d7f08c3d2c6a4a00b2e715a1b78R25
Or not sure what exactly is meant?

Reply:
Hey, sry for the late answer. 
You have removed the `:routerLink="{ name: 'sw.review.index' }` of the button and added the onCancel method with router push. Wasnt it the same behaviour with the router link?

Reply:
Unfortunatly I am now on holidays, so I will need to see how good I can modifications to this PR.

But yes that is correct, I replaced the link by the function which is needed because of the shortcut mapping. In my opinion this is not only a cosmetic change, but also unifies the behavior of the back button and the shortcut, as it is the case for example in the product detail view. I can change the link again, but the `onCancel` method would still be required.

Reply:
Tank you for the explanation. I'm okay with this change.